batch/batch/driver/instance_collection/job_private.py

```diff
@@ -308,7 +338,10 @@ async def create_instances_loop_body(self):
             should_wait = True
             return should_wait
         user_share = {
-            user: max(int(300 * resources['n_allocated_jobs'] / total + 0.5), 20)
+            user: min(
+                int(self.max_instances_per_autoscaler_loop * resources['n_allocated_jobs'] / total + 0.5),
+                resources['n_allocated_jobs'],
+            )
```

Comment:
Why did you change the semantics here? I thought this was just a PR to make these values configurable.

Reply:
It was originally to make the parameters configurable. But the current behavior wasn't correct as it would spin up a constant number of instances that wasn't configurable (I believe 300 and/ or 20 per user). I can make the change separately, but felt that it was appropriate here in the context of the overall goal to make these parameters configurable. I didn't want it to look like this parameter was configurable for job-private, but in reality it does nothing.

Your call on whether to make a separate PR.

Reply:
Actually, on second thought, the reason this code was like this in the first place was because the job private manager shouldn't care how quickly it ramps up as it's a job per worker. So 300 was the max for GCP rate limits. I think this change is still good though as it will be consistent with the pool.